src/products/src/products-service/routes.go

```diff
@@ -53,4 +53,16 @@ var routes = Routes{
 		"/products/category/{categoryName}",
 		CategoryShow,
 	},
+	Route{
+		"ProductUpdate",
+		"POST",
+		"/products/",
+		UpdateProduct,
+	},
+	Route{
+		"InventoryUpdate",
+		"POST",
+		"/products/inventory",
```

Comment:
Should the path be `/products/inventory/{productID}` so it's in the resource context of a product?

Reply:
corrected to 		"/products/inventory/{productID}",